app/models/spree/order_decorator.rb

```diff
@@ -0,0 +1,22 @@
+Spree::Order.class_eval do
+  checkout_flow do
+    go_to_state :address, :if => lambda { |order| order.address_required? } 
+    go_to_state :payment, :if => lambda { |order| order.payment_required? }
+    go_to_state :confirm, :if => lambda { |order| order.confirmation_required? }
+    go_to_state :complete
+  end
```

Comment:
Can you place a space between these two methods please

Reply:
@candanedo please remember to **always** add spaces between each method definition. It might not be obvious in this instance, but `checkout_flow` is treated as a method definition in this context.

Reply:
Of course